ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -100,36 +99,40 @@ const CancelSpeedupPopover = () => {
           alignItems={AlignItems.center}
           display={DISPLAY.FLEX}
           variant={TextVariant.bodySm}
-          as="h6"
-          marginTop={0}
-          marginBottom={2}
+          marginBottom={4}
```

Comment:
Should this be 2 here?

Reply:
Great catch @DDDDDanica! The previous component was adding a padding-bottom of 8px and a margin-bottom of 8px this simplifies it a bit by removing the css and using box props to create the same layout